.github/workflows/android.yml

```diff
@@ -37,23 +37,170 @@ jobs:
       run: touch local.properties
       working-directory: android
 
-    - name: Check formatting
-      run: ./gradlew ktfmtCheck
-      working-directory: android
-      
     - name: Build with Gradle
       env:
         GITHUB_ACTOR: ${{ github.actor }}
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: ./gradlew build
       working-directory: android
 
+  check-ktfmt:
```

Comment:
Is there any way we can reduce the amount of copypasta here? Either with a first-class GitHub actions concept or with YAML anchors etc.?